src/components/CategoriesTable/CategoriesTableRow.tsx

```diff
@@ -22,16 +22,19 @@ export const CategoriesTableRow = ({ category, formatting }: Props): ReactElemen
     <tr>
       <td className={formatting}>{category.id}</td>
       <td className={formatting}>{category.name}</td>
-      <td className={'text-center ' + formatting}>{category.products?.length ?? '0'}</td>
-      <td className={'flex justify-end ' + formatting}>
+      <td className={`text-center ${formatting}`}>{category.products?.length ?? '0'}</td>
+      <td className={`flex justify-end ${formatting}`}>
         <span
           className="group mr-4 flex h-[33px] w-[33px] cursor-pointer items-center justify-center rounded border bg-[#E4EAEB]"
           onClick={() => formContext.startEditing(category)}
+          onKeyDown={() => formContext.startEditing(category)}
```

Comment:
I'm still unsure about this. Eslint with Airbnb style guide wants a keyboard event when adding onClicks to non-interactive elements such as spans and divs.

Reply:
interestingly I don't see this rule specified in the guide sheet https://airbnb.io/javascript/react/

could this maybe be conflicting with something?

Reply:
actually it seems like its part of `jsx-a11y` https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/click-events-have-key-events.md, reading the reasoning for this rule does make a ton of sense but is this something we should be enforcing ourselves right now?

Reply:
@mbaltrimas maybe? Just so we can say we're thinking about 'accessibility'? Do you know if there's a way to implement this without using two events - onclick, onkeypress - or refactor it to look better?

Reply:
using semantic html would be one way I reckon - swap out the span for a button

Reply:
You can't have buttons within a button, which is what would have if the top layer was a button

Reply:
yeah true we can't swap out every element that has an onClick with an interactable element, but we could do it in places where it makes sense. The span above could be replaced, it already has its role set to "button" so it may as well be a button 😄 - but in terms of other elements, I'm not exactly sure if there is a way of avoiding onKeyDown without disabling the rule entirely 🤔